### DISCUSSION TITLE
现在mirai2.0是必须用openjdk吗？

### DISCUSSION THREAD

**Discussion by a08381** [mamoe/mirai#779] [Q&A] [answered]
自己遇到了一个问题，看了其他issue，里面有开发者提到要换成openjdk，换成openjdk之后确实好了，但是我几台开发机都是用的oracle jdk，想问一下以后会支持其他jdk吗？还是说锁死openjdk了呢？

**Comment by Him188** [comment, marked as answer]
OracleJDK 会检查 JCE 安全性，打包之后 BC 的签名失效，因此会报错

原生就支持其他 JDK，只是通常用户都会打包导致这个问题才不建议使用 OracleJDK

**Reply by Him188** [reply]
相关的更多问题：https://github.com/mamoe/mirai/discussions/852
